floor_annotation/lib/src/fts.dart

```diff
@@ -0,0 +1,34 @@
+class Fts {
```

Comment:
💡 FTS4 has some additional features that FTS3 doesn't support. Such as setting an external content table to be used by the FTS table. It might make sense to create two separate types: `FTS3` and `FTS4` as it's done in Room (see links below). We don't have to support all of these additional features initially but having two classes might allow us to be more flexible in the future.
Having two FTS classes also requires some changes to `Entity` because it would either need to understand both the `FTS3` and `FTS4`types or we list FTS out of the `Entity` annotation as done in Room. What are your thoughts about this?

https://developer.android.com/reference/androidx/room/Fts3
https://developer.android.com/reference/androidx/room/Fts4

Reply:
at last time ，i didn't known how to get a new annotation on classElement
i change to code like room now